packages/ui/src/components/Loader/dots.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState, useEffect, useRef, useCallback } from 'react';
+import { StyleSheet, Animated, Easing } from 'react-native';
+import Proptypes from 'prop-types';
+
+import { utils } from '@monkvision/toolkit';
+
+const { useNativeDriver } = utils;
+const styles = StyleSheet.create({
+  loading: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  },
+});
+
+/**
+ * @param dots
+ * @param colors
+ * @param dotSize
+ * @param bounceHeight
+ * @param borderRadius
+ * @return {JSX.Element}
+ * @constructor
+ */
+export default function Dots({ dots, colors, dotSize, bounceHeight, borderRadius }) {
+  const [animations, setAnimations] = useState([]);
+
+  const opacity = useRef(new Animated.Value(0)).current;
+
+  useEffect(() => {
+    const dotAnimations = [];
+    // eslint-disable-next-line no-plusplus
+    for (let i = 0; i < dots; i++) {
```

Comment:
Use `i += 1` instead of `++`. Don't disable eslint rules like that. ^^ Do you have de description in your Editor when you have a linting error like this ? Also, a fix file command should transform ++ operator to += 1. :-)